winch/filetests/filetests/x64/table/size.wat

```diff
@@ -0,0 +1,15 @@
+;;! target = "x86_64"
+(module
+  (table $t1 0 funcref)
+  (func (export "size") (result i32)
+    (table.size $t1))
+)
+;;    0:	 55                   	push	rbp
+;;    1:	 4889e5               	mov	rbp, rsp
+;;    4:	 4883ec08             	sub	rsp, 8
```

Comment:
This is a little surprising, shouldn't  we always grow the stack by multiples of 16? (I'm assuming we call some sysv64 functions at some point, which requires 16-byte alignment for the stack.)

Reply:
Currently calls follow the system ABI requirements when it comes to alignment, but the alignment is done at the callsite, since we don't try to maintain alignment at all times (your stack could go unaligned if/when you perform any spills). Also this alignment will most certainly change to 16 once we support SIMD, but nevertheless we might still end up with an unaligned stack and so we would still need to perform at-callsite alignment. Hope this clarifies things.